README.md

```diff
@@ -356,3 +367,30 @@ be uploaded to Google Drive first, and then the "Sync w/Google Drive" button wil
   ```
   RESOURCE_TYPE_FIELDS=resourcetype,filetype,<your_custom_field_name>
   ```
+
+# Enabling AWS MediaConvert transcoding
+
+The following environment variables need to be defined in your .env file:
+
+```
+AWS_ACCOUNT_ID
+AWS_ACCESS_KEY_ID
+AWS_SECRET_ACCESS_KEY
+AWS_STORAGE_BUCKET_NAME
+VIDEO_S3_TRANSCODE_ENDPOINT
+AWS_ROLE_NAME
+DRIVE_SHARED_ID
+DRIVE_SERVICE_ACCOUNT_CREDS
+API_BEARER_TOKEN
+```
+
+This will allow for videos to be submitted for transcoding to the AWS MediaConvert service. This is done automatically once a video has been synced to Studio from Google Drive.
+
+# Enabling 3Play integration
+
+The following environment variables need to be defined in your .env file (for a pre-configured 3Play account):
+
+```
+THREEPLAY_API_KEY
+THREEPLAY_CALLBACK_KEY
```

Comment:
We also have a `THREEPLAY_PROJECT_ID` whose value defaults to the production's project, so I think we should add that here too.